migrations/versions/0371_replace_template_content.py

```diff
@@ -0,0 +1,25 @@
+"""
+Revision ID: 371_replace_template_content
+Revises: 0370_notification_callback_url
+Create Date: 2024-10-08 20:35:38
+"""
+from alembic import op
+
+revision = '371_replace_template_content'
```

Comment:
This should match the file name to be consistent with the other migration files.
```suggestion
revision = '0371_replace_template_content'
```